File: backend/IndustrySyncApi/Models/WorkOrder.cs
namespace IndustrySyncApi.Models;

public class WorkOrder
{
    public long Id { get; set; }
    public string Title { get; set; } = string.Empty;
    public string Description { get; set; } = string.Empty;
    public string Priority { get; set; } = "Low";
    public bool IsCompleted { get; set; }
    public DateTime CreatedAt { get; set; } = DateTime.UtcNow;
}

File: app/src/main/java/com/example/industrysync/WorkOrder.kt
package com.example.industrysync

import com.google.gson.annotations.SerializedName

data class WorkOrder(
    @SerializedName("id") val id: Long = 0,
    @SerializedName("title") val title: String,
    @SerializedName("description") val description: String,
    @SerializedName("priority") val priority: String,
    @SerializedName("isCompleted") val isCompleted: Boolean = false
)